test/integration/crl_test.go

```diff
@@ -40,25 +35,25 @@ func runUpdater(t *testing.T) {
 func TestCRLPipeline(t *testing.T) {
 	// The crl-updater and crl-storer are not yet deployed in Prod, so only run
 	// this test in config-next.
-	if !strings.Contains(os.Getenv("BOULDER_CONFIG_DIR"), "config-next") {
+	configDir := os.Getenv("BOULDER_CONFIG_DIR")
```

Comment:
I think you still need to check if this is empty.

Reply:
If it's empty, then it doesn't contain "config-next", and we exit on line 40.